libs/ledger-live-common/src/mock/account.ts

```diff
@@ -108,6 +109,199 @@ export function genAccount(id: number | string, opts: GenAccountOptions = {}): A
             cacheTransactionInfoById: {},
           };
           break;
+        case "cardano":
+          (account as CardanoAccount).cardanoResources = {
+            delegation: {
+              status: true,
+              poolId: "45",
+              ticker: "ADA",
+              name: "Cardano",
+              rewards: new BigNumber(42),
+            },
+            externalCredentials: [],
+            internalCredentials: [],
+            utxos: [],
+            protocolParams: {
+              minFeeA: "",
+              minFeeB: "",
+              stakeKeyDeposit: "",
+              lovelacePerUtxoWord: "",
+              collateralPercent: "",
+              priceMem: "",
+              priceSteps: "",
+              languageView: {
+                PlutusScriptV1: {
```

Comment:
Ouch :sweat_smile: so there was no way to use an empty object here? I think all this stuff is only useful is you need to actually sign and broadcast a valid tx.

But anyway now that it's done no problem.